src/components/PopupWithDeleteCard.js

```diff
@@ -0,0 +1,22 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithDeleteCard extends Popup {
```

Comment:
Needs correction: this is a general class. It can be used for any confirmation, so there should not be mentions of `card`. Please rename this class to `PopupWithConfirmation`